tests/tests/bitasset_tests.cpp

```diff
@@ -542,6 +544,455 @@ BOOST_AUTO_TEST_CASE( hf_890_test )
    }
 }
 
+class bitasset_evaluator_wrapper : public asset_update_bitasset_evaluator
+{
+public:
+   void set_db(database& db)
+   {
+      this->trx_state = new transaction_evaluation_state(&db);
+   }
+};
+
+/******
+ * @brief Test various bitasset asserts within the asset_evaluator before the HF 922 / 931
+ */
+BOOST_AUTO_TEST_CASE( bitasset_evaluator_test_before_922_931 )
+{
+   // set up the basics for the test
+   fc::log_level log_level_all(fc::log_level::all);
+
+   BOOST_TEST_MESSAGE("Advance to near hard fork 922 / 931");
+   auto maint_interval = db.get_global_properties().parameters.maintenance_interval;
+   generate_blocks( HARDFORK_CORE_922_931_TIME - maint_interval);
+   trx.set_expiration( HARDFORK_CORE_922_931_TIME - maint_interval + fc::hours(1));
```

Comment:
Good, but the maximum expiration time is also configurable... >:-) maximum_time_until_expiration

Reply:
Fixed